routes/api/follows.js

```diff
@@ -4,24 +4,34 @@ const createError = require('http-errors');
 const { sequelize, Follow } = require("../../db/models");
 const {asyncHandler} = require('../utils');
 
+/* Delete a follower from a user */
+router.delete('/follows/:id', asyncHandler(async (req, res) => {
 
+        /* Initialize follow object */
+        const follow = await Follow.findPk(req.params.id);
 
+        /* Set status 404 if follow doesn't exist */
+        if (typeof follow === "undefined" ) {res.status(404)};
```

Comment:
`res.status` just sets the header of the response. Sending it using `res.statusCode(404)` is an option, but I think it's best we utilize error handlers here. We'll also need to make sure that the rest of the code does not attempt to execute, so be sure to return in the if statement.

To utilize error handlers, use next with an error similar to when `res.locals.authenticated` is `false`.

Reply:
Code has been changed to use next and use the error handler.